### DISCUSSION TITLE
problem with com connection

### DISCUSSION THREAD

**Discussion by Mima27** [luc-github/Repetier-Firmware-0.92#408] [F.A.Q] [answered]
good morning everyone I did the reset on pin j37 and now when I turn on the printer and connect it to the PC it no longer recognizes the COM port but a generic USB driver which does not allow me to proceed with the compilation of the Arduino
![Immagine WhatsApp 2023-09-18 ore 13 31 28](https://github.com/luc-github/Repetier-Firmware-4-Davinci/assets/91128553/a1be89d9-66af-46fe-a8ab-d301f63dc7d9)

**Comment by luc-github** [comment]
Did you installed due arduino module ? it should add the drivers also

if yes did you tried the manual update like suggested here : https://docs.arduino.cc/software/ide-v1/tutorials/getting-started/cores/arduino-sam

**Reply by Mima27** [reply]
I installed the drivers but in device manager it doesn't load the com port and every time I connect the printer it recognizes it as a generic USB

**Reply by luc-github** [reply, marked as answer]
did you update manually  like the doc describe ?

**Reply by Mima27** [reply]
Yes, it was installed manually as described in the instructions. Maybe tonight I'll try to switch to version 1.6.12 as written in the descriptions you sent me, while I had installed 1.6.8 as written previously. As soon as I do this test I'll update you. Thank you very much for the support.

**Reply by Mima27** [reply]
![16951444198864614254126235013573](https://github.com/luc-github/Repetier-Firmware-4-Davinci/assets/91128553/f3e45e79-0ff6-4bf8-949c-e09b4c7ea579)
![16951444902315415491266958471628](https://github.com/luc-github/Repetier-Firmware-4-Davinci/assets/91128553/5e75bcb5-e2a6-4c8e-9327-7df050c2d0e4)

I tried multiple versions of the library and also of Arduino but the problem remained as you can see the library is installed correctly. and the printer was reset with jumper j37

**Reply by Mima27** [reply]
![Uploading 16951466497081308262254963809920.jpg…]()

**Reply by Mima27** [reply]
Hi luc, is it possible to have remote support?

**Reply by luc-github** [reply]
Did the printer was recognized by windows before doing the J37 jumper ?
How many time did you reset the board with J37 ?
[drivers.zip](https://github.com/luc-github/Repetier-Firmware-4-Davinci/files/12684124/drivers.zip)

**Reply by Mima27** [reply]
the printer was recognized by win with the original firmware and it printed until last week, the reset was done a couple of times because I thought the problem was there but nothing always gave an unrecognized USB

**Reply by luc-github** [reply]
I have joined the arduino due drivers used on my PC - give a try to see if updating drivers using then works

**Reply by Mima27** [reply]
Can you provide me with the Arduino version you use and the driver version? so I try with the same ones as yours?

**Reply by luc-github** [reply]
Due module V1.6.8  with arduino ide V1.8.16
 the drivers are the same since 2015 I think

**Reply by Mima27** [reply]
Hi Luc, so I tried to install the Arduino IDE and the drivers on a new PC with the same versions as yours but even so the printer is not recognized which makes me think there is a problem with the board. Can you just confirm the reset way? with the printer off I make the bridge on pin 37 after a few seconds I turn off the printer and remove the bridge, turning it back on it should connect without firmware is this correct?

**Reply by luc-github** [reply]
as mentionned on Readme
1 - bridge on Pin 37 when machine is Power OFF
2 - Power ON when bridge is still active
3 - Wait few sec (5~10 sec ?)
4 - Power OFF
5 - Remove the bridge
6 - Power ON 
7 - The machine should be ready to be flashed

**Reply by Mima27** [reply]
Hi luc, finally with the 4 pc I managed to recognize the printer on the com port, unfortunately however it is recognized with the bossa program port drivers

**Reply by luc-github** [reply]
yes that is normal - it will be recognised as arduino due after first flash because bootloader will be updated

**Reply by Mima27** [reply]
OK perfect! I tried on Saturday but it didn't seem to load the sketch. I'll try again tonight

**Reply by luc-github** [reply]
here someone who did a video long time ago to show the bossa port :
https://youtu.be/rjuCvlnpB7M?t=256
It use arduino method - but I suggest to use the platformio one as I have added the patch on the process so in platformio the process is simpler: nothing to do but compile and flash
